### PR TITLE
added a progress-bar in event creation form

### PR DESCRIPTION
**Pull request by swekshajha12**
#135 
I added a progress bar and created a separate js file to keep the html file clearer.
progresss

### CODE REVIEW

File: vms/event/templates/event/progress.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   
+});
+
+$('#v8').change(function() {
+    valeur+=a8;
+    a8=0;
+     document.getElementById("myProgress").value = valeur;
+   
+   
+});
```
Comment (tapasweni-pathak) [inline vms/event/templates/event/progress.js]: Cool! ✨ 

Did you try with flow like fill the details, see the progress bar dancing forward and backward, then delete random things does it touch zero or does it go back? It didn't for me.

File: vms/event/templates/event/create.html [hunk 2]
```diff
@@ -198,17 +206,17 @@
                     </div>
                 {% else %}
                     <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "Venue" %}</label>
+                        <label class="col-md-2 control-label">Venue</label>
```
Comment (tapasweni-pathak) [inline vms/event/templates/event/create.html]: Also can you please reason the removal of ```trans``` VMS supports French, there we need this and to support other languages in future. 🚢

File: vms/event/templates/event/create.html [hunk 3]
```diff
@@ -1,7 +1,11 @@
+
 {% extends "administrator/settings.html" %}
 
-{% load i18n %}
```
Comment (tapasweni-pathak) [inline vms/event/templates/event/create.html]: woops! this shouldn't be removed. Can you please reason this removal?

File: vms/event/templates/event/progress.js (hunk 1 above)
Comment (swekshajha12) [reply vms/event/templates/event/progress.js]: Yes , it was working for me. I'll look it up again!

File: vms/event/templates/event/create.html (hunk 2 above)
Comment (swekshajha12) [reply vms/event/templates/event/create.html]: I'll do it!

File: vms/event/templates/event/progress.js (hunk 1 above)
Comment (tapasweni-pathak) [reply vms/event/templates/event/progress.js]: Did you get the chance to look it up again? It did not work for me now too. :(

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @swekshajha12 What about using this https://pypi.python.org/pypi/django-progressbarupload/0.1.5

**Comment by tapasweni-pathak** [comment]
Please reopen if you plan to pick this up again.

**Pull request closed without merging**